src/test/java/dk/cngroup/lentils/service/CypherServiceTest.java

```diff
@@ -73,12 +76,12 @@ public void checkCodeword() {
 
         when(repository.findByStage(TESTED_STAGE)).thenReturn(cypher);
 
-        assertTrue(service.checkCodeword(CODEWORD, TESTED_STAGE));
+        assertTrue(service.checkCodeword(generator.CODEWORD, TESTED_STAGE));
```

Comment:
Tady radsi prosim pouzij ObjectGenerator.CODEWORD, protoze je konvence, ze kdyz pristupujes ke staticke konstance, tak se pristupuje pres tridu a ne pres instanci. Jinak davam approve, takze si to pak muzes sama mergnout tady tento pullrequest ;)